test/test_nn.py

```diff
@@ -2157,6 +2157,14 @@ def forward(self, X):
                 Id = torch.eye(X.size(0), device=X.device)
                 return torch.linalg.solve(Id + X, Id - X)
 
+        class Resize(nn.Module):
+            def forward(self, X):
+                return X[[0]]
```

Comment:
What is this notation? I had never seen it before and a quick search does not throw anything meaningful.

Reply:
`X[[0]]` just selects the indices in the list inside the outside brackets i.e. the 0-th row